components/brave_rewards/browser/rewards_service_impl.h

```diff
@@ -189,6 +192,20 @@ class RewardsServiceImpl : public RewardsService,
   void GetAddressesForPaymentId(const GetAddressesCallback& callback) override;
   std::pair<uint64_t, uint64_t> GetEarningsRange();
 
+  void GetAutoContributeCount(
```

Comment:
can we create a bitwise enum (similar to chromium data removal) for this instead of adding individual methods?

Reply:
sorry, this was supposed to be on `RemoveAutoContributeData`

Reply:
Now have one method and bitwise operations will be handled in native ledger. Still question lingering about starting values for enum. Will address in DM.

Reply:
Done. bitwise checks are in ledger_client_ with datatypes defined in ledger